tests/tester.js

```diff
@@ -13,6 +13,7 @@ const RuleTester = getRuleTester()
 const eslintVersion = new Linter().version
 const ecmaVersion =
     /*eslint-disable prettier/prettier */
+    semver.gte(eslintVersion, "9.0.0") ? 2025 :
```

Comment:
https://github.com/eslint/eslint/commit/8824aa1469ffc572c5e5c1765d1b6da113dfba19 has not been released, I think it's fine to update to the upcoming version. :)
```suggestion
    semver.gte(eslintVersion, "9.6.0") ? 2025 :
```